spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :user do
-    email { 'random@randon.com' }
+    email { "#{rand(10000000000)}random@randon.com" }
```

Comment:
This feels a bit hacky. I would suggest you take a look at the Faker gem.

Reply:
Hacky indeed. We'll take a look